OxfordBodleian/Juel-Jensen/BDLaethf24.xml

```diff
@@ -162,9 +162,22 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                             </handNote>
                         </handDesc>
 
-                       <!-- <decoDesc>
+                       <decoDesc>
+                         <summary>Decoration on <locus target="#3r"/> could be contemporary with the making of the manuscript. That on <locus target="#4r"/>
+                       appears to be posterior.</summary>
+                         <decoNote type="ornamentation" xml:id="d1">
+                             <locus target="#3r"></locus>
+                             <desc>Monochrome unframed irregular interlace pattern painted in red and above the upper ruling margin and outlined with black ink.
```

Comment:
I don't understand the sentence well, "painted in red and above" - is "and" here necessary, or is there perhaps a word missing?

Reply:
No the subject "interlace pattern" is "painted in red" and "painted above the upper ruling margin"  I could delete the "and" to make it clearer :)

Reply:
Ah, now I get it!